cloudinit/config/schema.py

```diff
@@ -830,19 +859,25 @@ def _get_property_description(prop_config: dict) -> str:
     prop_descr = prop_config.get("description", "")
     oneOf = prop_config.get("oneOf", {})
     anyOf = prop_config.get("anyOf", {})
-    descriptions = [] if not prop_descr else [prop_descr.rstrip(".")]
+    descriptions = []
+    deprecated_descriptions = []
+    if prop_descr:
+        prop_descr = prop_descr.rstrip(".")
+        if not prop_config.get(DEPRECATED_KEY):
+            descriptions.append(prop_descr)
+        else:
+            deprecated_descriptions.append(_add_deprecation_msg(prop_descr))
```

Comment:
Thank you @aciba90 for unravelling my code additions to make this more legible in the future. This a good improvement